browser/ui/webui/settings/brave_wallet_handler.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_WALLET_HANDLER_H_
+#define BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_WALLET_HANDLER_H_
+
+#include "chrome/browser/ui/webui/settings/settings_page_ui_handler.h"
+
+class Profile;
+
+class BraveWalletHandler : public settings::SettingsPageUIHandler {
+ public:
+  BraveWalletHandler() = default;
+  ~BraveWalletHandler() override = default;
+
+ private:
+  // SettingsPageUIHandler overrides:
+  void RegisterMessages() override;
+  void OnJavascriptAllowed() override {}
+  void OnJavascriptDisallowed() override {}
+
+  void GetAutoLockMinutes(base::Value::ConstListView args);
+
+  DISALLOW_COPY_AND_ASSIGN(BraveWalletHandler);
```

Comment:
This is deprecated, we should stop using it in new classes.

Reply:
Oops copied from existing.

Reply:
rebased out here https://github.com/brave/brave-core/pull/10392/files#diff-76a0cf7b9026674ba12a29e20a1817d42fddf0b51d9f8496d6331c312ed78155R26